prow/gerrit/reporter/reporter_test.go

```diff
@@ -114,7 +137,7 @@ func TestReport(t *testing.T) {
 			},
 		},
 		{
-			name: "finished pj, no spec",
+			name: "finished pj",
```

Comment:
Please make the name include both the scenario and expected result. So "report when pj finishes" here

Reply:
done